test/SSHRemoting/SSHRemoting.Basic.Tests.ps1

```diff
@@ -54,6 +54,13 @@ Describe "SSHRemoting Basic Tests" -tags CI {
             VerifySession $script:session
         }
 
+        It "Verifies explicit Options parameter" {
+            $options = @{"Port"="22"}
+            $script:session = New-PSSession -HostName localhost -Options $options -ErrorVariable err
+            $err | Should -HaveCount 0
+            VerifySession $script:session
+        }
```

Comment:
This test checks that the parameter is only recognized, but does not verify that it is being applied.

Reply:
I can definitely add more integration tests, it will take me a few days to get those done though.